### DISCUSSION TITLE
Utilização do CorsConfig

### DISCUSSION THREAD

**Discussion by willyantomaz** [joaoagr1/rpg-manager-system-api-java#3] [Q&A] [answered]
Acredito que para uma aplicação que será libera para o acesso em um servidor publico futuramento, a ultilização do CorsConfig não seria o ideal já que vc também ultiliza o CrosOrigin para liberar o acesso, o que vc acha ?

**Comment by joaoagr1** [comment, marked as answer]
A utilização do CorsConfig pode não ser ideal, uma vez que já está utilizando CrossOrigin para liberar o acesso. No entanto, optei por manter o CrossOrigin devido a dificuldades na configuração do acesso do servidor na nuvem através do CorsConfig.
